internal/vm/setup.go

```diff
@@ -76,6 +85,19 @@ _G["load_plugin_schema"] = function(plugin_schema_string)
   return plugin_name, nil
 end
 
+_G["unload_plugin_schema"] = function(plugin_name)
+  local base_error = "error unloading schema for plugin: "
+  if plugin_name == nil or plugin_name == "" then
+    return false, base_error .. "base_error"
+  end
+
+  if Plugins and Plugins.subschemas and Plugins.subschemas[plugin_name] then
```

Comment:
Just curious: in which circumstances this part of the check may fail?

`if Plugins and Plugins.subschemas`